dubbo-config/dubbo-config-api/src/main/java/org/apache/dubbo/config/ServiceConfig.java

```diff
@@ -341,7 +333,15 @@ public synchronized void export() {
         }
 
         if (delay != null && delay > 0) {
-            delayExportExecutor.schedule(this::doExport, delay, TimeUnit.MILLISECONDS);

```

Comment:
can we call `delayExportExecutor.shutdown()` method after the delay task add work queue?

Reply:
Yes,This is another way to fix this .

Reply:
Don't think so.
If I want to export the service at runtime(use `ServiceConfig`), turning off `delayExportExecutor` will cause problems.

Reply:
> Don't think so.
> If I want to export the service at runtime(use `ServiceConfig`), turning off `delayExportExecutor` will cause problems.

you are right

Reply:
> Don't think so.
> If I want to export the service at runtime(use `ServiceConfig`), turning off `delayExportExecutor` will cause problems.

yes.